hedyorg/hedy
# 💻 New slides

Fixes #4660
Fixes #4932 
Fixes #5056 

I've removed the iFrame code, this way the code is easier to edit 

**How to test**
1. Go to `/for-teachers`, open a slides level
2. Run a program for which you would get an achievement and make sure no achievement is shown
3. Test the language switcher 
4. Test all slides, if code, text and headers are shown etc
